bazel/rules_def.bzl

```diff
@@ -28,6 +28,7 @@ def ptxla_cc_test(
             "-isystemexternal/torch",  # Required for system includes.
             "-fexceptions",  # Required for testing crashes.
         ],
+        hdrs_check = "strict",
```

Comment:
Is this an option defined in tf_cc_test? I don't see it documented in Bazel: https://bazel.build/reference/be/c-cpp

Reply:
From that document: `All header files that are used in the build must be declared in the hdrs or srcs of cc_* rules. This is enforced.`

Does that option have any effect in open-source bazel?

https://bazel.build/reference/be/c-cpp#hdrs

Reply:
This should be the default, but it does not work with gcc I think (there's a gazillion transitive imports in this project and it does not fail the build). I'll remove it though, we should move to clang when upstream issues are fixed with the cppmap declarative outputs.